pom.xml

```diff
@@ -198,8 +222,8 @@
       <plugin>
         <artifactId>maven-compiler-plugin</artifactId>
         <configuration>
-          <source>1.7</source>
-          <target>1.7</target>
+          <source>1.8</source>
```

Comment:
You could just remove the entire section and use `java.level` property

Reply:
Yes, remove all this.

Reply:
done